cni/network/network_windows.go

```diff
@@ -410,3 +410,14 @@ func (plugin *NetPlugin) isDualNicFeatureSupported(netNs string) bool {
 	log.Errorf("DualNicFeature is not supported")
 	return false
 }
+
+func getOverlayGateway(podsubnet *net.IPNet) (net.IP, error) {
+	ncgw := podsubnet.IP
+	ncgw[3]++
+	ncgw = net.ParseIP(ncgw.String())
```

Comment:
check this out: https://stackoverflow.com/a/31191775

Reply:
we're passing the IP from net.IPNet instead of podip